wasmtest.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+
+# wasmtest.sh is a shell script that can compile and run wasm tests.
+# There are three funtions for wasmteest.sh(1.test single file 2.test all files 3.test all tests from the file user give)
+# To test single file use: ./wasmtest.sh single <your file name>.c 
+# To compile single file use: ./wasmtest.sh singlecompile <your file name>.c
+# To run single file use: ./wasmtest.sh singlecompile <your file name>.c
+# To test all files use: ./wasmtest.sh all
+# To compile all files use: ./wasmtest.sh allcompile
+# To run all files use: ./wasmtest.sh allrun
+# To test all tests from the file user give use: ./wasmtest.sh files
+# To compile all tests from the file user give use: ./wasmtest.sh filescompile
+# To run all tests from the file user give use: ./wasmtest.sh filesrun
+
+glibc_base="/home/lind-wasm/src/glibc"
+wasmtime_base="/home/lind-wasm/src/wasmtime"
```

Comment:
I'd prefer to have a LIND_WASM_BASE variable to add flexibility to the path. For example, the LIND_WASM_BASE variable has a default value of `/home/lind-wasm`, but it can be overwriten by environment variable